build/NugetProperties.props

```diff
@@ -6,10 +6,10 @@
 
   <!-- Official Version -->
   <PropertyGroup>
-    <MajorVersion>2</MajorVersion>
-    <MinorVersion>2</MinorVersion>
+    <MajorVersion>3</MajorVersion>
```

Comment:
I think you have some old code in this branch. The new versions now exist in .csproj files of both projects.

Reply:
You're right. I rebased and am now using the new versioning placement.